src/main/java/it/pagopa/transactions/controllers/v1/TransactionsController.java

```diff
@@ -66,6 +71,15 @@ public Mono<ResponseEntity<ProblemJsonDto>> openStateHandler() {
                                 .detail("Upstream service temporary unavailable. Open circuit breaker."),
                         HttpStatus.BAD_GATEWAY
                 )
+        ).doOnNext(
+                ignored -> openTelemetryUtils.addErrorSpanWithAttributes(
+                        OpenTelemetryUtils.CIRCUIT_BREAKER_OPEN_SPAN_NAME
+                                .formatted(error.getCausingCircuitBreakerName()),
+                        Attributes.of(
+                                OpenTelemetryUtils.CIRCUIT_BREAKER_OPEN_NAME_ATTRIBUTE_KEY,
```

Comment:
redundant information? circuit breaker name is already present into error span name, since this will be used only to trace circuit breaker errors I would not add this attribute (and avoid add a new span attribute to the indexed ones on elastic). what do you think?

Reply:
Yes, you're right, I don't think we need to make an indexed search on circuit breaker name, so I remove the attribute

Reply:
Removed here https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/502/commits/69181d2294821fc0f7325cf2d857c8fc3a3d10be